engine/server/webapp/asset-manifest.json

```diff
@@ -1,13 +1,13 @@
 {
   "files": {
-    "main.css": "/static/css/main.7b6afa70.css",
-    "main.js": "/static/js/main.2f3b687c.js",
+    "main.css": "/static/css/main.1fada45f.css",
```

Comment:
can we nuke the previous css files? i see new files being added but the old ones aren't being removed!

also curious whether these assets should be in `.gitignore` if they are generated assets

Reply:
not yet; i have not added frontend to build pipeline so need these static files; will nuke it after doing so.. in the pipeline :D